homeassistant/components/websocket_api/commands.py

```diff
@@ -45,7 +45,6 @@ def handle_subscribe_events(hass, connection, msg):
 
     Async friendly.
     """
-    from .permissions import SUBSCRIBE_WHITELIST
```

Comment:
There is a reason this is here. It's for circular imports.

Reply:
Thanks for the heads up! Moved it back in https://github.com/home-assistant/home-assistant/pull/29556/commits/6a1e5a60742ab59e33c11f99c14c9919a252fa28